modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -62,6 +64,40 @@ rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0
 !condev rd.dasd=0.0.3490
 ----
 
+
+** For installations on FCP type disks, specify:
```

Comment:
Similar to the previous comment, it doesn't make sense to have the lead-in sentence end with "specify" if there aren't parallel actions to take in the numbered list that follows. So I suggest something like:

s/specify:/complete the following steps:

```
... Use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed. For multipathing, repeat this step for each additional path. 
... For multipathing, set the following parameter as `rd.multipath=default`
... For multipathing,  set the install device as `coreos.inst.install_dev=/dev/mapper/mpatha`
... For single-path installation, set the install device as `coreos.inst.install_dev=sda`
```